src/db.ts

```diff
@@ -11,6 +11,7 @@ import InviteModel from './lib/invite/model';
 import SubmissionModel from './lib/submission/model';
 
 import logger from './logger';
+import RepositoryModel from './lib/repository/repositoryModel';
```

Comment:
```suggestion
import InviteModel from './lib/invite/model';
import SubmissionModel from './lib/submission/model';
import RepositoryModel from './lib/repository/repositoryModel';

import logger from './logger';
```

![image](https://github.com/teach-hub/core-service/assets/44705155/31375221-cfb9-42c0-ac0c-0b36d1d00dff)